lib/src/runner/runtime_utils.dart

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2015, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// JS runtime files utilities used by dartdevrun.
+library dev_compiler.src.runner.runtime_utils;
+
+import 'dart:async';
+import 'dart:io';
+
+import 'file_utils.dart' show listJsFiles;
+import 'package:dev_compiler/src/options.dart';
+import 'package:path/path.dart';
+
+/// In node.js / io.js, these modules need to be aliased globally
+/// (e.g. `var foo = require('./path/to/foo.js')`).
+const _ALIASED_RUNTIME_FILES = const {
+  'dart_library.js': 'dart_library',
```

Comment:
another option here, we could change our module pattern, to make these imports explicit if we're in nodejs mode.
(in ES6 we'll want to do that too. The current DDC module pattern is a bit of a hack...)